### PR TITLE
Network requests

### LINKED ISSUE #3: Iter3: User Story
User Story:

- When the server returns a 500 error, the user will see proper error handling

Acceptance Criteria:
- the app should fetch data from the api endpoint for both a single movie and all the movies. 
- If an attempt to fetch data results in a 500 server error, user should be shown an error message reflecting the error (and, quite possibly, fiercely apologizing)

### PR DESCRIPTION
**Pull request by JoshMallery**
Hello Team!

What (if any) features are you implementing?
This PR adds in the api calls and fetch requests as well as does some additional CSS styling.

What (if anything) did you refactor?
We did refactoring where the data comes from.

Does this PR close any issues?
closes #3 

Happy Coding!

### CODE REVIEW

**Review by Abumpus1 — APPROVED** [review]
Nice to have error handling, we will need to make sure to replace the placeholder React image in the future. Good job!

File: src/App.js [hunk 1]
```diff
@@ -1,48 +1,42 @@
...
+      movie: [],
+      error:""
     };
   }
 
   componentDidMount() {
-    this.setState({
-      movies: movieData.movies
-    })
+    Promise.all([apiCalls.getMovies()])
```
Comment (robbiejaeger) [inline src/App.js:19]: I'm a little confused by the use of `Promise.all` here. I would think `getMovies` would return a single promise, which would make the use of `Promise.all` unnecessary, but I'll try to look into that.

File: src/apiCalls.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const apiCalls = {
+
+  getMovies: (id = "") => {
+  return  fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
```
Comment (robbiejaeger) [inline src/apiCalls.js:4]: Yes, so it looks like you're either getting an array of movies or a single movie with this function. Either way, you'll get one promise object back. So you don't need to be using `Promise.all` above.

File: src/components/MovieDetailContainer.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ import "./MovieDetailContainer.css";
 function MovieDetailContainer({movie}) {
   const {title,tagline,budget,revenue,overview,runtime,genres} = movie
 
-  const genresToDisplay = genres.map(genre=> <p>{genre} </p>)
+  const genresToDisplay = genres.map((genre,index)=> <p key={index}>{genre} </p>)
```
Comment (robbiejaeger) [inline src/components/MovieDetailContainer.js:7]: Indexes can be useful for using as a key, but keep in mind that they have caveats. Look into this section of the docs to learn more about when you don't want to use the index as a key: https://reactjs.org/docs/lists-and-keys.html#keys

**Pull request merged**